terraform/ggroup-to-splunk/lambda_iam.tf

```diff
@@ -4,10 +4,10 @@ data "template_file" "lambda_trust" {
 }
 
 data "template_file" "lambda_policy" {
-  template = file("${path.module}/json/interface_lambda/policy.json")
+  template = file("${path.module}/lambda_json/policy.json")
   vars = {
     region     = var.region
-    account_id = data.aws_ssm_parameter.account_id.value
```

Comment:
Why would you not use the `data` parameter in favour of a variable?

Reply:
This has since been removed